testing/regress/environment.xml.in

```diff
@@ -935,6 +944,10 @@
                           path="WsELKAccess"
                           resource="WsELKAccess"
                           service="ws_elk"/>
+    <AuthenticateFeature  description="Access to WS code signing service"
```

Comment:
Here (and everywhere else) I'm not sure the description is quite right - the flag affects (or will affect) other places besides the code signing service, and should say something like "Access to sign ECL code"

The resource name could also be changed to "CodeSignAccess"

Reply:
Changed description and renamed resource